### DISCUSSION TITLE
Can it deploy in Raspberry Pi

### DISCUSSION THREAD

**Discussion by Quinn-Liu** [ludwig-v/wireless-carplay-dongle-reverse-engineering#35] [Ideas]
I think raspberry pi is a very ideal device to achieve u project，is there any other reason to choose i.MX 6?
I know new NAC/RCC SoC is i.MX 6

I think the best deivce is raspberry pi zero wireless or CM4，or raspberry pi4 is much too big

**Comment by CiNcH83** [comment]
You will most probably need an Apple/MFi authentication chip. NXP/Freescale provide i.MX based MFi development kits.

**Reply by iDoka** [reply]
MFi chip is not an issue for usage RPi.
As external interface MFi chip has I2C bus - we would to communicate with it on RPi I think.

PS: I suppose we will might use unmodified binary from iMX6. AFAIK, I2C bus communication hardcoded in binary in such way: `/dev/i2c-X` - it seems we just need bring up RPi's I2C host on the same location in /dev.
Is it makes sense?

**Comment by hassankhan** [comment]
Hi @Quinn-Liu 

Some folks seem to be working on running something similar on a Raspberry Pi over at this repo: https://github.com/45clouds/WirelessCarPlay

**Comment by p0358** [comment]
Given the instructions about running this on a Mac from readme I'm wondering if it could be possible to run it like this on a Raspberry or any Linux device without Android and the APK for performance. Just open a browser without any DE or WM and have it render the page directly on the screen?

**Comment by typeRob** [comment]
Does somebody have any kind of update about this? I want to try to run it on the PI but if someone is interested in helping it would be easier...
